extensions/wikia/AppPromoLanding/crowdin.conf

```diff
@@ -1,5 +0,0 @@
-[crowdin.project]
-projectid = "ext-wikia-AppPromoLanding"
```

Comment:
Are we using this crowding project somewhere else? Should we remove it from crowdin as well?

Reply:
yes it's safe to be removed from crowdin, I will notify Keiko about it